core/test/functional/api/v2/admin/posts_spec.js

```diff
@@ -0,0 +1,359 @@
+const should = require('should');
+const supertest = require('supertest');
+const _ = require('lodash');
+const testUtils = require('../../../../utils');
+const localUtils = require('./utils');
+const config = require('../../../../../../core/server/config');
+const ghost = testUtils.startGhost;
+let request;
+
+describe('Post API', function () {
+    let ghostServer;
+
+    describe('As Owner', function () {
+        let ownerCookie;
+
+        before(function () {
+            return ghost()
+                .then(function (_ghostServer) {
+                    ghostServer = _ghostServer;
+                    request = supertest.agent(config.get('url'));
+                })
+                .then(function () {
+                    return localUtils.doAuth(request, 'users:extra', 'posts');
+                })
+                .then(function (cookie) {
+                    ownerCookie = cookie;
+                });
+        });
+
+        describe('Browse', function () {
+            it('retrieves all published posts only by default', function (done) {
+                request.get(localUtils.API.getApiQuery('posts/'))
+                    .set('Cookie', ownerCookie)
+                    .set('Origin', config.get('url'))
```

Comment:
@allouis This is how i used the cookie in the test. I just did it quickly, but it worked.

Reply:
:+1: That's how it's done!

Reply:
👍